index.html

```diff
@@ -105,6 +105,7 @@
       }
 
       .select-player-modal .modal-body {
+        position: relative;
```

Comment:
This is needed so `#close` takes its position from `modal-body` and not `modal-content`.